docs: Add link from `auto_https` option to Auto HTTPS doc (#76)
caddyserver/website: src/docs/markdown/caddyfile/options.md

	- **interval** and **burst** allows `<n>` certificate operations within `<duration>` interval.
- **local_certs** causes all certificates to be issued internally by default, rather than through a (public) ACME CA such as Let's Encrypt. This is useful in development environments.
- **key_type** specifies the type of key to generate for TLS certificates; only change this if you have a specific need to customize it.

> - **auto_https** configure automatic HTTPS. It can either disable it entirely (`off`) or disable only HTTP-to-HTTPS redirects (`disable_redirects`). Check [Automatic HTTPS](/docs/automatic-https) for details on how these options affects http/https support.

Reviewer: I think keeping it simple is fine here.

```suggestion
- **auto_https** configure automatic HTTPS. It can either disable it entirely (`off`) or disable only HTTP-to-HTTPS redirects (`disable_redirects`). See the [Automatic HTTPS](/docs/automatic-https) page for more details.
```

> - **auto_https** configure automatic HTTPS. It can either disable it entirely (`off`) or disable only HTTP-to-HTTPS redirects (`disable_redirects`). Be aware that if you choose `disable_redirects` or `off`, port :80 won't be open unless a rule explicitelly listens to it on ingress, for example, with `mysite.org:80`.

Reviewer: That's not exactly true. Caddy will still use `:80` (or more specifically, the value of the `http_port` global option) for ACME HTTP challenges. 

https://github.com/caddyserver/website/pull/74 should help clarify some of these things.

Rather, I think what could be changed here instead is to link to the Automatic HTTPS documentation page which better describes the behaviours.

Author: Great discussion on #74... PR #77 sure would have solved my doubts, specially with the following table:

![image](https://user-images.githubusercontent.com/7790172/89953041-c0bfda00-dc04-11ea-8452-751a138df3dd.png)
